### PR TITLE
Feature/telegrambot create event

### PR DESCRIPTION
**Pull request by hryesf**
useful document for more information about Telegram API :

https://core.telegram.org/bots/api

### CODE REVIEW

File: src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        if (update.hasMessage()) {
+            Message message = update.getMessage();
+
+            if (message.hasText()) {
+                String text = message.getText();
+
+                if (text.equals("/start")) {
+                    SendMessage sendMessage = new SendMessage();
+                    sendMessage.setText("\"به برنامه مدیریت ایونت JUG خوش آمدید \uD83D\uDC4B\uD83D\uDE0D\"");
+                    sendMessage.setParseMode(ParseMode.MARKDOWN);
```
Comment (hesamghiasi) [inline src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:37]: What is parse mode?

File: src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+                    try {
+                        execute(sendMessage);
+                    } catch (TelegramApiException e) {
+                        e.printStackTrace();
+                    }
+                }
+
+            }
+        } else if (update.hasCallbackQuery()) {
```
Comment (hesamghiasi) [inline src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:105]: What is call back query? Why we need to check it here?

**Review by hesamghiasi — COMMENTED** [review]
please find answer to the following questions

File: src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java (hunk 2 above)
Comment (hryesf) [reply src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:105]: > What is call back query? Why we need to check it here?

https://core.telegram.org/bots/api#callbackquery

Comment (hryesf) [reply src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:105]: In Telegram bots, a CallbackQuery is a type of message that represents a user's interaction with an inline keyboard. When a user presses a button in an inline keyboard, a callback query is generated and sent to the bot, containing information about the button that was pressed.

Callback queries are useful for creating interactive and dynamic bot interfaces, allowing users to trigger specific actions or provide input without having to send a separate message.

File: src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java (hunk 1 above)
Comment (hryesf) [reply src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:37]: > What is parse mode?

The three parse modes in Telegram bots, namely Markdown, MarkdownV2, and HTML, are used to format the text sent by the bot. Here's a breakdown of each parse mode:

Markdown: This parse mode allows you to format text using simple Markdown syntax. You can use asterisks (*) or underscores (_) to make text bold, and use backticks () to create inline code blocks. For example, bold text` will result in bold text.

MarkdownV2: This parse mode is an extended version of Markdown and provides additional formatting options. In addition to the features available in Markdown, you can use double underscores (__) to underline text and tilde (~) to create strikethrough text. For example, __underlined text__ will display as underlined.

HTML: This parse mode allows you to use HTML tags to format text. You can use tags like <b> for bold, <i> for italic, <u> for underline, and <s> for strikethrough. For example, <b>bold text</b> will show as bold text.

Comment (hryesf) [reply src/main/java/ir/tehran/jug/projects/eventmanagement/web/telegram/bot/EventManagerBot.java:37]: https://core.telegram.org/bots/api#formatting-options

**Comment by hryesf** [comment]
useful document to know about Telegram API :

https://core.telegram.org/bots/api

**Pull request merged**